test/odbc/odbc_handler.h

```diff
@@ -108,30 +91,25 @@ class OdbcHandler {
 
     void BindColumns(vector<tuple<int, int, SQLPOINTER, int>> columns);
 
+    map<ServerType, ConnectionStringObject> getOdbcDrivers();
+
   private:
     // ODBC-defined SQL variables
     SQLHENV henv_{};
     SQLHDBC hdbc_{};
     SQLHSTMT hstmt_{};
     RETCODE retcode_{};
-    
-    // DB Information
-    string db_driver_{};
-    string db_server_{};
-    string db_port_{};
-    string db_uid_{};
-    string db_pwd_{};
-    string db_dbname_{};
 
-    // Build the connection string for SQLDriverConnect
-    string connection_string_{};
 
     // A map that contains values from the configuration file
     map<string, string> config_file_values_{};
 
     // Goes through config.txt and returns a map with values from the configuration file
     map<string, string> ParseConfigFile();
 
+    // map of server type to connection string info
+    map<ServerType, ConnectionStringObject> odbc_drivers{};
```

Comment:
I agree, it's better I think to store this info in another class rather than the `OdbcHandler` class itself. And each `OdbcHandler` object will deal with one connection string.

Reply:
Done. Please review.